src/test/scala/no/ndla/conceptapi/TestData.scala

```diff
@@ -104,72 +129,4 @@ object TestData {
     status = Status.default
   )
 
-  val domainConcept_toDomainUpdateWithId = domain.Concept(
```

Comment:
Jeg klarer ikke heeeelt å se hvilken verdi det gir å fjerne disse for så å bruke tomme concepts i feks controller testene (vil nesten påstå at det er eeever so sliiiiightly mer verdifult at de har innhold), men har ikke så mye å si for meg.